frontend/storage/utils/createGameAbilitiesListData.ts

```diff
@@ -0,0 +1,37 @@
+import { AbilityExample } from '../models/AbilityExample';
+import { Ability } from '../models/Ability';
+import { AbilityGroup } from '../models/AbilityGroup';
+import { AbilityCategory } from '../models/AbilityCategory';
+import { GameAbilitiesListProps } from '../../components/GameAbilitiesList';
+
+interface GameAbilitiesListInitialProps {
+	abilityExamples: Array<AbilityExample>;
+	abilities: Map<number, Ability>;
+	abilityGroups: Map<number, AbilityGroup>;
+	abilityCategories: Map<number, AbilityCategory>;
+	gameId: number;
+}
+export function createGameAbilitiesListData(props: GameAbilitiesListInitialProps): GameAbilitiesListProps {
+	const { abilities, abilityExamples, abilityGroups, abilityCategories, gameId } = props;
+	const gameAbilitiesListData: GameAbilitiesListProps = { abilities: new Map(), categories: new Array() };
```

Comment:
From my experience unless you need to create an array with specific type (for TS purposes) it's most common to just use `[]` to initialize an empty one.